_format/1.1/index.md

```diff
@@ -585,6 +596,96 @@ that includes a [profile alias][profile aliases]:
+creating, updating or deleting resources](https://jsonapi.org/format/1.1/#crud). However, this is not a strict
+limitation on their use.
+
+In the example below, two links are established. A `self` link indicating the
+URL of the request that generated the current collection of articles and an `add`
+link indicating the that the client can add an article to this collection.
+
+```json
+"links": {
+  "self": {
+    "href": "http://example.com/articles",
+    "rel": ["self", "add"]
+  }
+}
+```
+
+In the next example, a `comment` link is a member of a `comments` [relationship
+object][relationships] and that relationship object is a member of an `article`'s relationships
+object. In this case, the `rel` member establishes two links indicating that
+the client can create a new comment at the target URL and that requests to the
+target `href` can create a relationship to the article's `comments` field.
+
+```json
+"links": {
+  "self": "http://example.com/articles/1/relationships/comments",
+  "related": "http://example.com/articles/1/comments",
+  "comment": {
+    "href": "http://example.com/articles/1/comments",
+    "rel": ["add", "relate"]
+  }
+}
+```
+
+In this example, a `self` link is a member of an `article`'s links object. In
+this case, the `rel` member established two links indicating that the client can
+edit, but not delete, the context article (perhaps another request with different
+authentication credentials would receive an additional link with a `remove` link
+relation type).
+
+```json
+"links": {
+  "self": {
+    "href": "http://example.com/articles/1",
+    "rel": ["self", "update"]
+  }
+}
+```
+
+#### <a href="#document-links-link-parameter-object" id="document-links-link-parameter-object" class="headerlink"></a> Link parameter objects
+
+"Link parameter objects" appear in link objects to represent a link's target
+attributes.
+
+A link parameter object **MAY** contain any of the following members: `hreflang`,
+`media`, `title`, and `type`. The values of these members **MUST** conform to their
+meanings as defined by [RFC8288 Section 3.4.1](https://tools.ietf.org/html/rfc8288#section-3.4.1).
+
+A link parameter object **MAY** contain other members as target attributes. Any
+additional member names **MUST** be valid target attribute names as defined by
+[RFC8288 Section 2.2](https://tools.ietf.org/html/rfc8288#section-2.2).
+
+Link parameter object members with multiple values **MUST** be represented with an
+array (i.e. a target attribute with multiple values should be an array of
+values, not a whitespace-separated string).
+
+A link parameter object **SHOULD NOT** contain a `rev` member.
+
+In order to represent a link with reversed semantics, it is **RECOMMENDED** that an
+alternate link relation type be used or, less preferably, that the `anchor`
+and `href` members be interchanged.
+
+> Note: The `rev` link parameter was deprecated by [RFC8288 Section 3.3](https://tools.ietf.org/html/rfc8288#section-3.3)
```

Comment:
Aha, this explains it. To me, `rev` triggers `revision`, not `reverse`. I think it'd be clearer to first have the `In order to represent […] reversed […]` paragraph, followed by the `SHOULD NOT […] rev`. Then the `Note: […]` would also be positioned more closely and the reader would be left wondering for less time.